content/en/security/application_security/enabling/compatibility/serverless.md

```diff
@@ -6,20 +6,38 @@ type: multi-code-lang
 code_lang_weight: 90
 ---
 
-<div class="alert alert-info">ASM support for AWS Lambda is in beta. Threat detection is done by using the Datadog's lambda extension.<br><br> Don't see your desired technology listed here? Datadog is continually adding additional support. Fill out <a href="https://forms.gle/gHrxGQMEnAobukfn7">this short form to send details</a>.</div>
 
-## Language and framework compatibility
-
-### Supported cloud environments
-
-- AWS Lambda (beta)
+## Language and ASM compatibility
+- AWS Lambda
+- Google Cloud Run
+- Azure App Service
+
+### AWS Lambda
+Notes
+- Threat Protection via Remote Config is not supported. However you can use Workflows to block IPs in your WAF instead.
+  
+|Type           	| Threat Detection	|  Vulnerability Management for OSS	| Vulnerability Management for Code-Level 	|
+| ---  		|   ---             		|           ----           			|           ----            				|
+| Java  		| {{< X >}}         	| 	                			|						|
+| .NET    	| {{< X >}}         	|  	                          		|						|
+| Node 		| {{< X >}}     		| beta	              			|						|
+| Python   	| {{< X >}}         	| beta                 			|						|
+| Ruby   	| {{< X >}}         	|  	                 			|						|
+| PHP   	| 		        	|	            			|						|
+| Go   		| {{< X >}}         	| 	               			|						|
+
+#### Supported trigger types
+ASM Threat Detection supports HTTP requests as function input only, as it is the highest likelihood of attackers exploiting a serverless application. HTTP requests typically come from AWS services such as:
+- Application Load Balancer (ALB)
+- API Gateway v1 (Rest API)
+- API Gateway v2 (HTTP API)
+- Function URL
 
-### Version dependencies
+<div class="alert alert-info">If you would like to see support added for any of the unsupported capabilities, let us know! Fill out <a href="https://forms.gle/gHrxGQMEnAobukfn7">this short form to send details</a>.</div>
 
-- Lambda Extension version: `39`
-- Serverless plugin version: `5.20.0`
+Get started with ASM monitored on your services →
```

Comment:
```suggestion
Get started with ASM monitoring on your services →
```

Reply:
I elected just to remove this sentence as it seemed redundant